opentracing-shim/src/main/java/io/opentelemetry/opentracingshim/SpanBuilderShim.java

```diff
@@ -179,16 +179,14 @@ public SpanBuilder withStartTimestamp(long microseconds) {
 
   @Override
   public Span start() {
+    io.opentelemetry.trace.Span.Builder builder = tracer.spanBuilder(spanName);
 
-    io.opentelemetry.trace.Span.Builder builder;
     if (ignoreActiveSpan && parentSpan == null && parentSpanContext == null) {
-      builder = tracer.spanBuilderWithExplicitParent(spanName, null);
+      builder.setParent((io.opentelemetry.trace.Span) null);
```

Comment:
You can now avoid caching the parent and redirect directly to the OpenTelemetry builder calls. Probably in a separate PR.

Reply:
Well, I've jumped from code entirely redirecting call to this, and not sure we will even keep this behavior now ;) (for example, if we have to support Out-of-band Spans, we might need to change the code again).

I will do such change later on, once we are sure things won't change as much ;)